README.md

```diff
@@ -1,3 +1,15 @@
-# Team-179-Backend
+## Team-179-Backend farmers crowdfunding app codebase
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/2b6c54d9481f4dbb86b5cf0b5ec7cdcf)](https://app.codacy.com/gh/BuildForSDGCohort2/Team-179-Backend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDGCohort2/Team-179-Backend&utm_campaign=Badge_Grade_Settings)
\ No newline at end of file
+Agri-fund, Agri-loop, Agri-vesty leverages networks for greater reach and exposure to collectively raise capital for farmers through investors.
+
+**General functionality:**
+
+-  Authenticate farmers, admin & investors (Login & Logout)
```

Comment:
Please checkout the indentation specifications from codacy as you continue editing

Reply:
Okay, done with issue #2. Check it out now